Frontend/src/views/MyFriends.vue

```diff
@@ -14,6 +14,11 @@ import FriendsList from "../components/friendsmanagement/FriendsList";
 import SearchNewFriend from "../components/friendsmanagement/SearchNewFriend";
 export default {
   name: 'MyFriends',
-  components: {FriendsList, SearchNewFriend}
+  components: {FriendsList, SearchNewFriend},
+  data() {
+    return {
+      myFriends: 'friends.myFriends',
+    }
+  }
```

Comment:
Kann raus, wenn myFriends in der jeweiligen Zeile eingefügt wurde

Reply:
Done